lldb/test/API/commands/frame/diagnose/dereference-function-return/TestDiagnoseDereferenceFunctionReturn.py

```diff
@@ -10,7 +10,7 @@
 
 
 class TestDiagnoseDereferenceFunctionReturn(TestBase):
-    @skipUnlessDarwin
+    @expectedFailureAll(oslist=no_match(lldbplatformutil.getDarwinOSTriples()))
```

Comment:
This test previously could not run anywhere but Darwin but now can and fails anywhere that isn't Darwin, correct?

Reply:
Yep